lib/resources/tasks/jest.js

```diff
@@ -13,9 +13,9 @@ export default (cb) => {
 
   process.env.BABEL_TARGET = 'node';
 
-  jest.runCLI(options, [path.resolve(__dirname, '../../')]).then((result) => {
-    if (result.numFailedTests || result.numFailedTestSuites) {
-      cb(new PluginError('gulp-jest', { message: 'Tests Failed' }));
+  jest.runCLI(options, [path.resolve(__dirname, '../../')]).then(({ results }) => {
+    if (results.numFailedTests || results.numFailedTestSuites) {
+      cb(new PluginError('jest-cli', { message: 'Tests Failed' }));
```

Comment:
It looks like (existing code) `new PluginError` generated excessive output in the console. To my limited understanding of gulp, it seems this is not a place for PluginError because it's not within context of a plugin.

Following gulpjs's example, `cb(new Error('Tests Failed'));` is enough and probably the correct way. @JeroenVinke can you have a look?

Reply:
I have hardly any experience with Gulp so left it in place, I agree the output seems excessive but it resolved the issue which was more important in my case. I will replace it with an error in my local setup and test to see if it also forces a build failure (I assume it does). If it needs to be replaced please let me know and I will update the PR.

Reply:
BTW, it looks jest is only feature in cli that uses plugin-error as a dep. If we decide to get rid of PluginError here, we can cleanup the `dependencies.json` and `unit-test-runners/jest.js` too.

Reply:
When using `new Error('...')` the build fails as expected:
![image](https://user-images.githubusercontent.com/3629094/52996750-cdec7600-341e-11e9-9021-40a0029a2f55.png)

@huochunpeng I think we can replace the PluginError.

Reply:
Today I also noticed just returning a string in the callback is enough to make it show an error.

Reply:
> Today I also noticed just returning a string in the callback is enough to make it show an error.

That's even better! Let's just clean up everything.

Reply:
@huochunpeng Updated and removed plugin-error package.